Packs/ThinkstCanary/Integrations/ThinkstCanary/ThinkstCanary_test.py

```diff
@@ -0,0 +1,23 @@
+import demistomock as demisto
+
+
+MOCK_PARAMS = {
+    'access-key': 'fake_access_key',
+    'secret-key': 'fake_access_key',
+    'server': 'http://123-fake-api.com/',
+    'unsecure': True,
+    'proxy': True
+}
+
+
+def test_fetch_incidents(mocker, requests_mock):
```

Comment:
just add the given when then docstring with the 
2020-01-07-07:58:18 -> 2020-01-07-07:58:19